cmd/kops/create_cluster.go

```diff
@@ -94,6 +97,21 @@ func (c *CreateClusterCmd) Run(args []string) error {
 		return err
 	}
 
+	if c.Manifest != "" {
+		clusterConfig, err := getManifest(c.Manifest)
+		if err != nil {
+			return err
+		}
+		// This is a WeakDecode() that won't throw an error if a directive does not exist.
+		clusterConfig.Unmarshal(c)
+		c.Zones = clusterConfig.GetString("Zones")
+		if c.Zones == "" {
```

Comment:
The zones directive currently supports **YAML strings** and **YAML lists**.. If a list is defined the GetString() function will return an empty string, we use that to then look for a YAML list with GetStringSlice()